tests/scripts/run-xunit-perf.cmd

```diff
@@ -94,9 +94,7 @@ if DEFINED TEST_ENV (
     )
 )
 
-xunit.performance.run.exe %BENCHNAME%.%TEST_FILE_EXT% -runner xunit.console.netcore.exe -runnerhost corerun.exe -verbose -runid %PERFOUT% > %BENCHNAME%.out
-
-xunit.performance.analysis.exe %PERFOUT%.xml -xml %XMLOUT% > %BENCHNAME%-analysis.out
+corerun.exe PerfHarness.dll %WORKSPACE%\sandbox\%BENCHNAME%.%TEST_FILE_EXT% --perf:runid Perf > %BENCHNAME%.out
```

Comment:
Yes, because we just append that at the beginning so that the files are easy to find.  They will all be Perf-<filename>.